src/components/cart.js

```diff
@@ -152,13 +155,27 @@ export default class Cart extends Component {
     // return this.props.client.fetchRecentCart();
   }
 
+  fetchMoneyFormat() {
+    return this.props.client.fetchShopInfo().then((res) => {
+      return res.moneyFormat;
+    });
+  }
+
   /**
    * initializes component by creating model and rendering view.
    * Creates and initalizes toggle component.
    * @param {Object} [data] - data to initialize model with.
    * @return {Promise} promise resolving to instance.
    */
   init(data) {
+    if (!this.config.moneyFormat) {
+      this.fetchMoneyFormat().then((moneyFormat) => {
+        this.moneyFormat = moneyFormat;
+        return moneyFormat;
+      }).catch((err) => {
+        return err;
```

Comment:
this is totally a hack. the linter was yelling at me so i just shut it up with this catch LOL
i'll fix it.